hack/verify-golint.sh

```diff
@@ -52,7 +52,7 @@ fi
 
 export IFS=$'\n'
 all_packages=(
-	$(go list -e ./... | egrep -v "/(third_party|vendor|staging|generated|clientset_generated)" | sed 's/k8s.io\/kubernetes\///g')
+	$(go list -e ./... | egrep -v "/(third_party|vendor|staging|generated|clientset_generated)" | sed -r "s/^_${KUBE_ROOT//\//\\/}\///g")
```

Comment:
This is a heck of a regex.

First, can you change to | to it's more like `s|^_${KUBE_ROOT//\//\\/}\/||g` ?  Reduce some noise :)

Second, what is the `_` for in there?

Third this doesn't match - go list produces strings like "k8s.io/kubernetes/test/utils" - no KUBE_ROOT there.